src/sections/main.tsx

```diff
@@ -1,10 +1,10 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
-import './index.css'
-import App from './App.tsx'
+import '../css/index.css'
+import App from '../components/App.tsx'
```

Comment:
we need to use absolute imports https://dev.to/smitterhane/avoid-relative-path-import-hell-in-react-36in

Reply:
This is fixed